finalProject/tenSpeckledFrogs.js

```diff
@@ -0,0 +1,20 @@
+var frogs = {
```

Comment:
Same feedback as Fizzbuzz - how could this be refactored to a function rather than a class?